### DISCUSSION TITLE
Lossless rips of game soundtracks in flac or similar formats possible in future versions or forks of ogg winmm?

### DISCUSSION THREAD

**Discussion by blam666** [ayuanx/ogg-winmm#21] [Q&A] [answered]
I know that it's probably not a feature deemed relevant or interesting to many, as the vintage cd game soundtracks that are targeted by this tool often didn't even have recordings of a quality that it should be notable to most users. But personally I would be quite delighted if I could make ogg-winmm use lossless rips in FLAC, WAV or similar formats to not experience any possible loss of quality through compression of OG CD soundtracks. I don't think a fork of ogg-winmm exists as of yet that is capable of using other, lossless formats (I may be wrong here, feel free to correct me). 

So I wanted to ask if there is any technical issue that makes it impossible to use anything else but lossy formats (I know of similar tools using MP3 format, but no single one providing lossless compatibility). Or is there simply no interest in providing such a feature?

**Comment by ayuanx** [comment, marked as answer]
There is no technical difficulty, just (in my opinion) pointless, but you don't have to agree with me.

First of all, CD-DA itself is not a high fidelity format in modern standard. The 16-bit @ 44KHz sample rate of CD-DA is too low to be considered as high fidelity. Even the audio in DVD which is 16-bit @ 48KHz is better than CD-DA, not to mention the modern high fidelity audio recorded as 24-bit @ 96KHz or even higher 24-bit @ 192KHz.

Secondly, you already have the freedom to set the ogg bitrate to your heart's content. I don't think there is a single person in this universe who can tell the difference between 320KB/s ogg and CD-DA. At least up to this day, not a single person in a double-blind experiment has succeeded in distinguishing them. There are already a lot of experiment reports carried out by professional audio studios that confirm it.

So at least for now, I don't have any incentive to support it.

**Reply by blam666** [reply]
Ok I see.

Thanks a lot for the reply!

**Comment by MIvanchev** [comment]
Just wanna chime in that OGG already supports FLAC streams, but IDK how hard it is to parse them.

**Comment by ayuanx** [comment]
> Just wanna chime in that OGG already supports FLAC streams, but IDK how hard it is to parse them.

Feel free to experiment, the parsing is done by OGG lib, so we don't need to do anything.
